test/rule_test.dart

```diff
@@ -180,15 +180,15 @@ void defineRuleUnitTests() {
 }
 
 /// Handy for debugging.
-void defineSoloRuleTest(String ruleToTest) {
-  for (var entry in Directory(ruleTestDataDir).listSync()) {
-    if (entry is! File || !isDartFile(entry)) continue;
-    var ruleName = p.basenameWithoutExtension(entry.path);
-    if (ruleName == ruleToTest) {
-      testRule(ruleName, entry);
-    }
-  }
-}
+// void defineSoloRuleTest(String ruleToTest) {
```

Comment:
Let's just delete this altogether? I think `rule_debug.dart` has superseded this approach.

Reply:
Thanks!